langchain/src/callbacks/tracers.ts

```diff
@@ -1,4 +1,3 @@
-import * as process from "process";
```

Comment:
@agola11 `process` is a global in node, and injected by bundlers in edge/browser/etc envs